ansible/roles/stack-sunbird/templates/ml-core-service.env

```diff
@@ -73,4 +73,8 @@ TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC={{ml_core_timezone_diffrenece_bet
 
 
 #Validate Entities enable/Disable
-VALIDATE_ENTITIES={{ml_service_validate_entities | default ("ON")}}
\ No newline at end of file
+VALIDATE_ENTITIES={{ml_service_validate_entities | default ("ON")}}
+
+#Public base url of sunbird
+PUBLIC_BASE_URL={{ml_core_public_base_url | default("")}}
```

Comment:
What should be the value of ml_core_public_base_url ? @ankitpws

Reply:
For staging  ml_core_public_base_url -  https://staging.sunbirded.org
For Dev ml_core_public_base_url -  https://dev.sunbirded.org   @gandham-santhosh